packages/react-query/src/__tests__/useInfiniteQuery.type.test.tsx

```diff
@@ -206,6 +216,13 @@ describe('getNextPageParam / getPreviousPageParam', () => {
       const result: Expect<
         Equal<InfiniteData<string> | undefined, (typeof infiniteQuery)['data']>
       > = true
+      // TODO: Order of generics prevents this from working
+      // const result: Expect<
+      //   Equal<
+      //     InfiniteData<string, number> | undefined,
```

Comment:
can we keep these commented in with an assertion towards:

```
InfiniteData<string, unknown>
```

?

Reply:
https://github.com/TanStack/query/pull/5811/commits/d881d1397ac3bd7a83d7a129555501af3af60441